Add CONTRIBUTIONS.md (#388)
PureStake/moonbeam: CONTRIBUTIONS.md

# ![Moonbeam](media/moonbeam-cover.jpg)

### Generated Documentation

> You can explore our crate-level documentation at

Reviewer: ```suggestion
You can explore our [crate-level documentation](https://purestake.github.io/moonbeam).
```

> [https://purestake.github.io/moonbeam](https://purestake.github.io/moonbeam). This documentation is

Reviewer: ```suggestion
This documentation is
```

# ![Moonbeam](media/moonbeam-cover.jpg)

## Contributions

> Moonbeam primarily uses GitHub Pull Requests to orchestrate code changes. If you wish to propose a

Reviewer: ```suggestion
Moonbeam primarily uses GitHub Pull Requests to coordinate code changes. If you wish to propose a
```
Just a personal preference, feel free to ignore

Author: Lol, fair enough
